src/software/world/robot_state.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include "software/new_geom/angle.h"
+#include "software/new_geom/angular_velocity.h"
+#include "software/new_geom/point.h"
+#include "software/util/time/timestamp.h"
+#include "software/world/robot_capabilities.h"
+
+class RobotState final
```

Comment:
Can we add a class comment here?

Reply:
done

Reply:
Can we add a comment describing what RobotState is? For example, RobotState represents the position, velocity, orientation, and angular velocity of a robot at a particular point in time.

Reply:
Perhaps a little more generic then that, just because at that point you're basically just repeating the class members?

Reply:
Perhaps

"This class represents the state of a Robot at a single point in time"?

Reply:
:+1: I like that.